pkg/commands/serviceversion/lock.go

```diff
@@ -48,7 +49,7 @@ func NewLockCommand(parent argparser.Registerer, g *global.Data) *LockCommand {
 // Exec invokes the application logic for the command.
 func (c *LockCommand) Exec(_ io.Reader, out io.Writer) error {
 	serviceID, serviceVersion, err := argparser.ServiceDetails(argparser.ServiceDetailsOpts{
-		AllowActiveLocked:  true,
+		Locked:             optional.Of(false),
```

Comment:
This is one of the actual bug fix changes: `fastly service-version lock` will now fail if the specified service-version is locked.